OpenRA.Mods.Common/Traits/World/CrateSpawner.cs

```diff
@@ -18,22 +18,25 @@ namespace OpenRA.Mods.Common.Traits
 {
 	public class CrateSpawnerInfo : ITraitInfo
 	{
-		[Desc("Minimum number of crates")]
+		[Desc("Minimum number of crates.")]
 		public readonly int Minimum = 1;
 
-		[Desc("Maximum number of crates")]
+		[Desc("Maximum number of crates.")]
 		public readonly int Maximum = 255;
 
-		[Desc("Average time (seconds) between crate spawn")]
+		[Desc("Average time (seconds) between crate spawn.")]
 		public readonly int SpawnInterval = 180;
 
+		[Desc("Delay (in seconds) before the first crate spawns.")]
+		public readonly int InitialSpawnDelay = 300;
```

Comment:
Can we have both of these in ticks to match our conventions in most other traits?